### ISSUE TITLE
Example COS service with binding not working

### ISSUE THREAD

**Issue by tpolekhin** [IBM/cloud-operators#45]
https://github.com/IBM/cloud-operators/blob/master/config/samples/cos.yaml

Message: `Failed: No deployment found for service plan lite at location us-south. Valid location(s) are: ["global"].`

Object description:
```
Name:         b2c-auth-dev-cos-dal-01-test
Namespace:    tpol
Labels:       <none>
Annotations:  kubectl.kubernetes.io/last-applied-configuration:
                {"apiVersion":"ibmcloud.ibm.com/v1alpha1","kind":"Service","metadata":{"annotations":{},"name":"b2c-auth-dev-cos-dal-01-test","namespace":...
API Version:  ibmcloud.ibm.com/v1alpha1
Kind:         Service
Metadata:
  Creation Timestamp:  2019-09-04T12:02:39Z
  Generation:          1
  Resource Version:    16278396
  Self Link:           /apis/ibmcloud.ibm.com/v1alpha1/namespaces/tpol/services/b2c-auth-dev-cos-dal-01-test
  UID:                 e4a0f379-cf0b-11e9-a426-e27ade6abd72
Spec:
  Plan:           lite
  Service Class:  cloud-object-storage
Status:
  Context:
    Org:
    Region:
    Resourcegroup:
    Resourcelocation:
    Space:
  Message:             Failed: No deployment found for service plan lite at location us-south. Valid location(s) are: ["global"].
Use service instance example if the service is a Cloud Foundry service
  Plan:                lite
  Service Class:       cloud-object-storage
  Service Class Type:
  State:               Failed
Events:                <none>
```

**Comment by vazirim** [comment]
@tpolekhin thanks for reporting this bug. We have a fix which will be in v0.1.1.
We are currently finalizing the new version.

**Comment by vazirim** [comment]
@tpolekhin This bug is now fixed in v0.1.1
The cos sample should now be written as follows:
```
apiVersion: ibmcloud.ibm.com/v1alpha1
kind: Service
metadata:
  name: mycos
spec:
  plan: standard
  serviceClass: cloud-object-storage
  context:
    region: global
```
